components/iam_app_user/main.tf

```diff
@@ -4,6 +4,10 @@ variable "name" {
 
 resource "aws_iam_user" "aws_user" {
   name = var.name
+
+  # When destroying this resource, destroy even if it has non-Terraform
+  # managed access keys, policies, etc. See: https://git.io/JfP7O
+  force_destroy = true
```

Comment:
Despite being named "force destroy", I do not believe this will force destruction on its own (it just forces it along if it already is going to destroy the resource for another reason). This is similar to a setting we have to use on our [Fastly properties](https://github.com/DoSomething/infrastructure/blob/fe6fac17c57d1179261a6333d2753483b389b9cf/dosomething/fastly-backend/main.tf#L23) to ensure that Terraform can delete them if it needs to.

I'll apply this change to Longshot, dev, and QA properties first to double-check that assumption! 🌵